precompiles/avsTask/parser.go

```diff
@@ -0,0 +1,39 @@
+package task
+
+import (
+	"fmt"
+
+	exocmn "github.com/ExocoreNetwork/exocore/precompiles/common"
+	types "github.com/ExocoreNetwork/exocore/x/avstask/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	cmn "github.com/evmos/evmos/v14/precompiles/common"
+	"golang.org/x/xerrors"
+)
+
+func (p Precompile) GetTaskParamsFromInputs(_ sdk.Context, args []interface{}) (*types.RegisterAVSTaskReq, error) {
+	if len(args) != 3 {
+		return nil, fmt.Errorf(cmn.ErrInvalidNumberOfArgs, 3, len(args))
+	}
+	taskParams := &types.RegisterAVSTaskReq{}
+	taskinfo := &types.TaskContractInfo{}
+
+	taskaddr, ok := args[0].(string)
+	if !ok {
```

Comment:
Also need to check if taskaddr == nil

Reply:
fixed

Reply:
The parameters are all strings, and using=="" can determine whether they are empty